node/src/components/small_network.rs

```diff
@@ -575,10 +575,13 @@ where
         add_to_blocklist: bool,
     ) -> Effects<Event<P>> {
         if let Some(incoming) = self.incoming.remove(&peer_id) {
+            trace!(%peer_id, "removing peer from the incoming connections");
```

Comment:
We want `our_id` in all log messages so we can filter on that.  I've found it's exceptionally useful when dealing with logs from more than one node.

Reply:
You're right, it would follow the already established pattern. At the same time, `our_id` is logged in like 50 places in this file already…

Reply:
Done in [`af4ffe3` (#738)](https://github.com/CasperLabs/casper-node/pull/738/commits/af4ffe3d128db7e6ca26bdb73da6cc90cfb79a27)